### PR TITLE
NFT support

### PR DESCRIPTION
**Pull request by belyakove**
PR adds NFT support and multiple changes to match Java implementation (swap, signed order etc) in https://github.com/zksync-sdk/zksync-java/pull/7.

### CODE REVIEW

File: ZKSyncSDK/Classes/Model/Swap/Order.swift [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+    func encode(to encoder: Encoder) throws {
+        var container = encoder.container(keyedBy: CodingKeys.self)
+        
+        try container.encode(accountId, forKey: .accountId)
+        try container.encode(recepientAddress, forKey: .recepientAddress)
+        try container.encode(nonce, forKey: .nonce)
+        try container.encode(tokenBuy, forKey: .tokenBuy)
+        try container.encode(tokenSell, forKey: .tokenSell)
+        try container.encode(ratio.0, forKey: .ratio)
+        try container.encode(ratio.1, forKey: .ratio)
```
[the remark this answers is not in the document]
Comment (MaximAlien) [reply ZKSyncSDK/Classes/Model/Swap/Order.swift]: Was updated to be encoded as an array, which contains both values of the tuple.

File: ZKSyncSDK/Classes/Signer/ZKSync/ZkSigner.swift [hunk 2]
```diff
@@ -77,79 +77,159 @@ public class ZkSigner {
...
-        data.append(try Utils.feeToBytes(transfer.feeInteger))
-        data.append(Utils.nonceToBytes(transfer.nonce))
-        data.append(Utils.numberToBytesBE(transfer.timeRange.validFrom, numBytes: 8))
-        data.append(Utils.numberToBytesBE(transfer.timeRange.validUntil, numBytes: 8))
-
-        let signature = try self.sign(message: data)
-        mutableTransfer.signature = signature
-        return mutableTransfer
+        transfer.signature = try compileSignature {
+            0x05
```
[the remark this answers is not in the document]
Comment (MaximAlien) [reply ZKSyncSDK/Classes/Signer/ZKSync/ZkSigner.swift]: Order was updated by @belyakove to match implementation in this [PR](https://github.com/zksync-sdk/zksync-java/pull/7).

**Comment by MaximAlien** [comment]
@StanislavBreadless, I think your previous comments were addressed. @belyakove has also made changes, which were required to match implementation in this [PR](https://github.com/zksync-sdk/zksync-java/pull/7). Can we get another round of review/approval?

**Review by StanislavBreadless — APPROVED** [review]
lgtm

**Pull request merged**